test/container_app_test.py

```diff
@@ -75,7 +75,24 @@ async def test_container_snapshot_restore(container_client, tmpdir, servicer):
 
 
 def square(x):
-    pass
+    return x**2
+
+
+@pytest.mark.asyncio
+async def test_container_snapshot_restore_stale_credentials_in_stub(container_client, tmpdir, servicer):
+    app = App()
+    from modal import Function
+    from modal.runner import deploy_app
+
+    app.function()(square)
+    app_name = "my-app"
+    deploy_app(app, app_name, client=container_client).app_id
+    f = Function.lookup(app_name, "square", client=container_client)
+    await f.remote.aio()
+    # RPC should have used new credentials, not old credentials
+    await f.remote.aio()
```

Comment:
i don't fully understand this test. where do you simulate the actual snapshot+restore?